src/metabase/models/comment.clj

```diff
@@ -11,13 +12,43 @@
   "Schema enum of the acceptable values for the `commented_item_type` column"
   (s/enum "moderation_request" "moderation_review"))
 
+(def commented-item->model
+  "Maps DB name of the commented item type to the model symbol (used for db/select and such)"
+  {"moderation_request" 'ModerationRequest
+   :moderation_request  'ModerationRequest
+   "moderation_review"  'ModerationReview
+   :moderation_review   'ModerationReview})
```

Comment:
It's possible that keywordizing the column was a mistake

Reply:
Will this map keep expanding in this way?  At some point, I presume it would be easier to add an `(if (string? x) (keyword x))` conversion somewhere?